server/public/css/bootstrap-overrides.css

```diff
@@ -1,3 +1,8 @@
 .bg-primary {
   background-color: #006400 !important;
+}
+.avatar {
```

Comment:
Is this a bootstrap class? If not can you move it to a different style sheet? Something like `custom.css` or `main.css`